libs/website/api/src/lib/schemas/article.graphql

```diff
@@ -23,6 +30,9 @@ fragment ArticleWithoutBlocks on Article {
 
   socialMediaDescription
   socialMediaTitle
+  socialMediaAuthors {
```

Comment:
Not used by any newsrooms right now, I'm working on getting that removed and other socialMedia/seo attributes